iocage_lib/ioc_create.py

```diff
@@ -657,11 +659,13 @@ def create_install_packages(self, jail_uuid, location, config,
                 silent=supply_msg[1])
             cmd = ("pkg", "install", "-q", "-y", pkg)
 
-            pkg_stdout, pkg_stderr, pkg_err = iocage_lib.ioc_exec.IOCExec(
+            pkg_stdout, pkg_stderr, pkg_errmsg = iocage_lib.ioc_exec.IOCExec(
                 cmd, jail_uuid, location, plugin=self.plugin,
                 silent=self.silent, msg_err_return=True).exec_jail()
 
-            if pkg_err and not self.plugin:
+            pkg_err = pkg_errmsg or pkg_err
```

Comment:
I am struggling to find the reasoning you need to have another var (pkg_errmsg).

Would only placing pkg_err = None at the top handle it, or am I missing something?